internal/view/exec.go

```diff
@@ -49,6 +49,9 @@ func runK(a *App, opts shellOpts) bool {
 	if g, err := a.Conn().Config().ImpersonateGroups(); err == nil {
 		args = append(args, "--as-group", g)
 	}
+	if isInsecure := a.Conn().Config().Flags().Insecure; *isInsecure {
```

Comment:
@verydragon Thank you for this PR. Great catch indeed! Better check for that potential nil here!

Reply:
Of course, nil check added. Thanks : )

Reply:
Awesome. Thank you!